sdk/src/types/block/output/native_token.rs

```diff
@@ -247,6 +251,13 @@ impl NativeTokens {
     }
 }
 
+// TODO: should we also impl `WorkScore` for `NativeToken` for plain consistency?
+impl WorkScore for NativeTokens {
+    fn work_score(&self, work_score_params: WorkScoreStructure) -> u32 {
+        self.len() as u32 * work_score_params.native_token
```

Comment:
todo, and yes id say so, even though its a bit pointless atm its better do have consistent imo